frontend/src/layouts/ProjectLayout.vue

```diff
@@ -11,6 +11,12 @@
       {{ $t("project.mode.tenant") }}
     </span>
   </h1>
+  <BBAttention
+    v-if="project.id == DEFAULT_PROJECT_ID"
+    class="mx-6 mb-4"
+    :style="`INFO`"
```

Comment:
This style attribute might doesn't works.

Reply:
Done

Reply:
At first I think it's the `style` attribute, but it's actually a component's prop :(